daemon/src/oracle.rs

```diff
@@ -2,7 +2,6 @@ use crate::db;
 use crate::model::cfd::CfdEvent;
 use crate::model::cfd::Event;
```

Comment:
a missing "not" in the commit description?

```
Remove spawn_fallible

Pretty much superseded by either Tasks::add or Tasks::add_fallible,
depending on the context. Tasks not tied to any lifetime are likely *not* what
we want.
```

Reply:
Reads as intended to me?

Reply:
I added the *not* in my version - not in the master branch? we don't want tasks tasks that are not tied to any lifetime, right?